Source/WebCore/platform/graphics/SourceBufferPrivate.h

```diff
@@ -125,7 +126,9 @@ class SourceBufferPrivate
 
     void setMediaSourceDuration(const MediaTime& duration) { m_mediaSourceDuration = duration; }
 
-    bool isBufferFullFor(uint64_t requiredSize, uint64_t maximumBufferSize);
+    WEBCORE_EXPORT virtual bool isBufferFullFor(uint64_t requiredSize) const;
+    WEBCORE_EXPORT virtual bool canAppend(uint64_t requiredSize) const;
+    virtual SourceBufferEvictionData evictionData() const { return m_evictionData; }
```

Comment:
Why is it virtual?

Reply:
so that the SourceBufferPrivateRemote can return a thread-safe version of it.

It had this in an earlier version, and it ended up not needed after I made `SourceBufferPrivate::EvictCodedFrame` return a boolean indicating if the buffer was full or not.

It removed the need for the SourceBufferPrivateClient to query the eviction data.

Reply:
removed